roles/openshift_etcd_migrate/templates/validatemigration.py.j2

```diff
@@ -0,0 +1,77 @@
+from subprocess import PIPE, Popen
+import logging
+import json
+import base64
+
+#logging.basicConfig(level=logging.INFO)
+
+def runCommand(cmd):
+	process = Popen(cmd, stderr=PIPE, stdout=PIPE, shell=True)
+	stdout, stderr = process.communicate()
+	rt = process.returncode
+
+	return stdout, stderr, rt
```

Comment:
Remember that returning ``stdout`` and``stderr`` will be either ``str`` (Python 2) or ``bytes`` (Python 3).